content/W2016/W16-event-stephanie-talk.md

```diff
@@ -0,0 +1,35 @@
+Title: Out of Town Speaker: Stephanie Morillo
```

Comment:
This is better I think, and I mentioned the name of the talk in the body of the event description